### PR TITLE
IngestArch: Add config example for Logstash multi-tenant arch

### PR DESCRIPTION
**Pull request by karenzone**
Fixes: #218 

**Preview:** https://ingest-docs_333.docs-preview.app.elstc.co/guide/en/ingest/master/ls-multi.html

### CODE REVIEW

**Comment by karenzone** [comment]
@jsvd @flexitrev FYI:  I'm working on a sample config for the Logstash-to-multiple-destinations architecture.

File: docs/en/ingest-arch/13-multitenant.asciidoc [hunk 1]
```diff
@@ -16,7 +16,31 @@ These logs need to be sent to different {es} clusters and to S3 for backup and a
...
-// ToDo: Add config example/logic in Logstash for routing data to different destinations
+Sample config::
++ 
+[source,ruby]
+input {
+  elastic_agent {
+    port => 5044
+  }
+}
+output {
```
Comment (karenzone) [inline docs/en/ingest-arch/13-multitenant.asciidoc:27]: @jsvd I'd like to hold the config to its most basic elements for clarity.  Is there's a filter (or filters) that would most likely be needed for this example to make sense (mutate maybe)? If so, I'll add it.

File: docs/en/ingest-arch/13-multitenant.asciidoc [hunk 2]
```diff
@@ -16,7 +16,31 @@ These logs need to be sent to different {es} clusters and to S3 for backup and a
...
+    cloud_id => "<cloud id>"
+    api_key => "<api key>"
+  }
+   s3 {
+     access_key_id => "my_key" 
+     secret_access_key => "my_access_key"
+     bucket => "our_bucket"
+   }
+  }
+}
```
Comment (jsvd) [inline docs/en/ingest-arch/13-multitenant.asciidoc:42]:
```suggestion
input {
  elastic_agent {
    port => 5044
  }
}
filter {
  translate {
    source => "[http][host]"
    target => "[@metadata][tenant]"
    dictionary_path => "/etc/conf.d/logstash/tenants.yml"
  }
}
output {
  if [@metadata][tenant] == "tenant01" {
    elasticsearch {
      cloud_id => "<cloud id>"
      api_key => "<api key>"
    }
  } else if [@metadata][tenant] == "tenant02" {
    elasticsearch {
      cloud_id => "<cloud id>"
      api_key => "<api key>"
    }
  }
}
```

**Review by karenzone — COMMENTED** [review]
Builds cleanly and renders as expected

**Comment by karenzone** [comment]
43247

**Pull request merged**